src/app/personside/infotabs/saksoversikt/saksdokumenter/JournalPostListeElement/JournalpostLestAvBruker.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import EtikettGraa from '../../../../../../components/EtikettGraa';
+import { Journalpost, Kommunikasjonsretning } from '../../../../../../models/saksoversikt/journalpost';
+import { formatterDatoTid } from '../../../../../../utils/date-utils';
+
+interface Props {
+    journalpost: Journalpost;
+}
+
+function JournalpostLestAvBruker({ journalpost }: Props) {
+    if (journalpost.retning !== Kommunikasjonsretning.Ut) {
+        return null;
+    }
+
+    return (
```

Comment:
Om `lestDato === null` så vil vi fortsatt spytte ut en del html her tror jeg.
Noe alà; 
```html
<div>
  <p>
    <div></div>
  </p>
</div>
```

Så tenker vi kan legge inn en liten guard for det, og forenkle html'en samtidig ;)